### PR TITLE
#2451 include non-polling feature in What's New

### PR DESCRIPTION
**Pull request by micheleRP**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2451
Review deadline: May 27

## Page previews
[What's New in 24.1](https://deploy-preview-518--redpanda-docs-preview.netlify.app/current/get-started/whats-new/#improved-fetch-latency) (we don't update for patch releases like 23.5)

## Checks

- [X] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/get-started/pages/whats-new.adoc [hunk 1]
```diff
@@ -32,6 +32,10 @@ Redpanda now prioritizes xref:manage:cluster-maintenance/cluster-balancing.adoc[
 == Managed identities for Azure
 Redpanda xref:manage:tiered-storage.adoc#tabs-1-microsoft-absadls[Tiered Storage] now supports https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/overview[managed identities for Azure] resources, similar to the existing support for IAM roles for self-hosted deployments on AWS and GCP. This provides a more secure alternative to static credentials, like access keys. Managed identities allow Redpanda clusters deployed on Azure virtual machines and on Azure Kubernetes Service (AKS) to access Azure Blob Storage (ABS) and other services without storing credentials in the cluster configuration.
 
+== Improved fetch latency
+
+The new cluster configuration property xref:reference:properties/cluster-properties.adoc#fetch_read_strategy[`fetch_read_strategy`] sets new clusters with a non-polling strategy to fulfill fetch requests. The backend is signaled when a partition has new data, so it doesn't need to repeatedly read from every partition in the fetch. This improves fetch latency when there is extra CPU space.
```
Comment (Feediver1) [inline modules/get-started/pages/whats-new.adoc]:
```suggestion
A new cluster configuration property, xref:reference:properties/cluster-properties.adoc#fetch_read_strategy[`fetch_read_strategy`], sets new clusters with a non-polling strategy to fulfill fetch requests. The backend is signaled when a partition has new data, so it doesn't need to repeatedly read from every partition in the fetch. This improves fetch latency when there is extra CPU space. 
```

**Review by Feediver1 — APPROVED** [review]

File: modules/get-started/pages/whats-new.adoc [hunk 2]
```diff
@@ -32,6 +32,10 @@ Redpanda now prioritizes xref:manage:cluster-maintenance/cluster-balancing.adoc[
 == Managed identities for Azure
 Redpanda xref:manage:tiered-storage.adoc#tabs-1-microsoft-absadls[Tiered Storage] now supports https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/overview[managed identities for Azure] resources, similar to the existing support for IAM roles for self-hosted deployments on AWS and GCP. This provides a more secure alternative to static credentials, like access keys. Managed identities allow Redpanda clusters deployed on Azure virtual machines and on Azure Kubernetes Service (AKS) to access Azure Blob Storage (ABS) and other services without storing credentials in the cluster configuration.
 
+== Improved fetch latency
+
+A new cluster configuration property, xref:reference:properties/cluster-properties.adoc#fetch_read_strategy[`fetch_read_strategy`], sets new clusters with a non-polling strategy to fulfill fetch requests. The backend is signaled when a partition has new data, so it doesn't need to repeatedly read from every partition in the fetch. This improves fetch latency when there is extra CPU space.
```
Comment (ballard26) [inline modules/get-started/pages/whats-new.adoc]: It should improve fetch latency and CPU utilization for most workloads. There are a few exceptions though, hence the reason for us retaining the old fetch strategy.

Comment (micheleRP) [reply modules/get-started/pages/whats-new.adoc]: Thanks @ballard26! I edited the last sentence. Do we need to document the exceptions for when users might not want the non-polling strategy?

Comment (jason-da-redpanda) [reply modules/get-started/pages/whats-new.adoc]: my suggested edits. i can do a seperate pr as realise im a bit late 
---
new cluster configuration property, xref:reference:properties/cluster-properties.adoc#fetch_read_strategy[`fetch_read_strategy`], which by default sets new clusters with a non-polling strategy to fulfill fetch requests. 
When polling fetch read strategy is enabled , the  redpanda fetch backend is signaled when a partition has new data, so it doesn't need to repeatedly read from every partition in the fetch. 
This improves fetch latency for the majority of workloads when there is extra CPU space available.
 
If your environment/workload was to experience any degradation of fetch latency after upgrading to 24.1.x  it is possible to revert to the previous strategy of "polling" via  command :
  
` rpk cluster config set fetch_read_strategy  polling`

Comment (micheleRP) [reply modules/get-started/pages/whats-new.adoc]: Thanks @jason-da-redpanda! Please check my revised section!

**Review by jason-da-redpanda — APPROVED** [review]

**Pull request merged**